deployments/polygon/wmatic/configuration.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Compound WMATIC",
+  "symbol": "cWMATICv3",
+  "baseToken": "WMATIC",
+  "baseTokenAddress": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
+  "baseTokenPriceFeed": "0xAB594600376Ec9fD91F8e885dADF0CE036862dE0",
+  "borrowMin": "1e18",
+  "governor": "0xCC3E7c85Bb0EE4f09380e041fee95a0caeDD4a02",
+  "pauseGuardian": "0x8Ab717CAC3CbC4934E63825B88442F5810aAF6e5",
+  "storeFrontPriceFactor": "TBD",
+  "targetReserves": "TBD",
+  "rates": {
+    "supplyKink": 0.9,
+    "supplySlopeLow": 0.025,
+    "supplySlopeHigh": 3,
+    "supplyBase": 0,
+    "borrowKink": 0.9,
+    "borrowSlopeLow": 0.035,
+    "borrowSlopeHigh": 3.5,
+    "borrowBase": 0.005
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "34722222222",
+    "baseBorrowSpeed": "0",
+    "baseMinForRewards": "TBD"
```

Comment:
`10000e18` - means, that COMP incentivization turns on when supplies/borrowed more than 10000 of Matic

Reply:
[5e19cb9](https://github.com/compound-finance/comet/pull/906/commits/5e19cb96f7cbc51ab8d588668107f4897126edcf)